### ISSUE TITLE
The properties panel does not appear in the "floating window" and "search""

### ISSUE THREAD

**Issue by cordinGH** [Macavity/siyuan-database-properties-panel#22]
The current properties panel can only be displayed when the document is open
But it can't be displayed in "search" or "floating windows".

**Comment by Macavity** [comment]
Yes, those views are not part of the document window and this plugin won't try to change them. Unless you want to propose to do this yourself?

**Comment by cordinGH** [comment]
> Yes, those views are not part of the document window and this plugin won't try to change them. Unless you want to propose to do this yourself?

是的，我

> Yes, those views are not part of the document window and this plugin won't try to change them. Unless you want to propose to do this yourself?

Thank you for clarifying! I'm not familiar with development, so I might not be able to implement this feature myself. Would it be possible to consider this as a future enhancement for the plugin?

**Comment by Macavity** [comment]
Ah these days with ChatGPT anyone can be a developer :) 

Sure, I think it makes sense to keep this around as a potential enhancement. But I don't have endless amounts of time on my hands and I don't personally see a lot of priority for this 🤷‍♂️ 
So, if this is important to you, you might be worth to consider doing it yourself.
